src/CMakeLists.txt

```diff
@@ -173,11 +179,11 @@ elseif(${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
     # GCC's arry bounds, maybe uninitialized, and restrict warning seems to have false positives.
     target_compile_options(lexy_dev INTERFACE -Wno-array-bounds -Wno-maybe-uninitialized -Wno-restrict)
 elseif(MSVC)
-    target_compile_options(lexy_dev INTERFACE /WX /W3 /D _CRT_SECURE_NO_WARNINGS /wd5105)
+    target_compile_options(lexy_dev INTERFACE /WX /W3 /EHsc /D _CRT_SECURE_NO_WARNINGS /wd5105)
```

Comment:
Please revert this change; the dev target is for internal use only where it doesn't matter.

Reply:
I got some error in some config, now I can not repreduce it anymore, will remove.